tests/integration_tests/canal_json_basic/data/data.sql

```diff
@@ -0,0 +1,124 @@
+USE `test`;
```

Comment:
It looks that this one can replace the case `CanalJson` in the GitHub action. 

Can we remove that one?

Reply:
Good catch! removed.

Thanks!